monitor-data/configure-monitors.mdx

```diff
@@ -6,6 +6,38 @@ tags:
     ['axiom documentation', 'documentation', 'axiom', 'dashboards', 'monitors', 'notifiers', 'slack', 'email', 'pagerduty', 'webhook', 'opsgenie', 'discord', 'discord webhook']
 ---
 
+## View monitor status
+
+To view the status of a monitor:
+1. Click the Monitors tab.
+1. Click the monitor in the list whose status you want to view.
+
+The monitor status page provides an overview of the monitor’s current status and history.
+
+On the left, you see the recent activity and the history of the monitor runs:
+- The `_time` field displays the time of the monitor run.
+- The `Status` field displays the status of the monitor.
+- The `Range from` and `Range to` fields display the time range used in the monitor run.
+
+You can change the time range of this overview in the top right corner.
+
+On the right, you see details about the monitor’s configuration.
+- Current status
+- Monitor type
+- Query the monitor periodically runs
+- Configuration details
```

Comment:
This reads funny to me, since it appears in a list of things that we've referred to as 'details about the monitor's configuration'

Reply:
Thanks! Corrected.